test/integration/facet.js

```diff
@@ -0,0 +1,72 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: loopback-workspace
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+const async = require('async');
+const app = require('../../');
+const expect = require('../helpers/expect');
+const fs = require('fs-extra');
+const testSupport = require('../helpers/test-support');
+const loopback = require('loopback');
+const Facet = app.models.Facet;
+const WorkspaceManager = require('../../lib/workspace-manager');
+
+describe('Facet', function() {
+  describe('CRUD', function() {
+    let workspace, dir;
+    before(function(done) {
+      testSupport.givenBasicWorkspace('empty-server', function() {
+        workspace = WorkspaceManager.getWorkspace();
+        dir = workspace.getDirectory();
+        done();
+      });
+    });
```

Comment:
http://loopback.io/doc/en/contrib/style-guide.html#layout-of-test-files -- see the bad section